### DISCUSSION TITLE
【Keyball61_右側トラックボール有り】左端の中央キーに離れたキーを押すと離れた２キーの内容で入力される

### DISCUSSION THREAD

**Discussion by JinBallMen** [Yowkees/keyball#299] [Q&A]
以下の３キー間で問題が起きており、問い合わせしました。
似たような問い合わせがあるかも知れませんが、若干症状が違うと思いますのでご容赦ください。

左端の中央のキーで、Shift + 7が入力されてしまいます。本来は別の入力がされるかと思っています。
左端中央の入力については、以下の箇所でも同じ入力がされているようです（左端中央入力時のOLEDは、②キーが入力されているような表示でした)
①Shift→左端の中央キーの下
②７→右から３列目の一番上

試した事は以下ですが、他に確認すべき事があればご教示いただきたいです。
・Pro Microの差し替え　→　変化なしでした
・基板側のPro Micro差し込み側のはんだ見直し　→　素人目での確認ですが、COL3の一部にはんだが乗らなくなってしまっている以外の問題は見られません
・印字が分からなくなったダイオードの取替　→　変化なしでした
　すべてを確認したわけではないですが、少なくとも関係のある箇所のダイオードの向きは、基盤の目印線と合っていると考えています
・問題箇所のPCBソケットのはんだ見直し、交換　→　変化なしでした

![20230806_123003~2](https://github.com/Yowkees/keyball/assets/75989686/3e94a412-0e11-436a-861b-07f5f92439af)

**Comment by JinBallMen** [comment]
Remapから見るとShift+7が割り当てられていたので、正常動作のようです。お騒がせしました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

キーマップの問題でしたか。直ってよかったです！
